template/script.sh.erb

```diff
@@ -20,7 +20,14 @@ module load xalt/latest <%= context.cuda_version %>
 cd "${NOTEBOOK_ROOT}"
 
 # Setup Jupyter environment
-module load project/ondemand <%= context.version %>
+# If we choose ascend, we only want to load the app_jupyter module, instead of app_jupyter/$VERSION$ module
+
+<% if context.cluster.match?(/ascend/) -%>
+  module load project/ondemand app_jupyter
```

Comment:
Should probably still load version so that users can still pick their version.  Lmod modules have a version associated, so even if there is only one version available, can still load by version.  Having a version option and it doing nothing is going to cause a lot of confusion for users who ask for something and get something different.

Reply:
@treydock   This is fixed.  As well have updated dynamic choice options in the form.